sdk/eventhub/Azure.Messaging.EventHubs.CheckpointStore.Blob/Directory.Build.props

```diff
@@ -0,0 +1,38 @@
+﻿<Project ToolsVersion="Current" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--
+    Identify samples as a test support project to keep the target framework synchronized, since both are
+    based on a version of netcoreapp.
+
+    Because the common SDK build properties need this value in order to recognize the desired
+    target frameworks and related, this needs to be set before the common properties import.
+  -->
+  <PropertyGroup Condition="$(MSBuildProjectName.EndsWith('.Samples'))">
+    <IsTestProject>true</IsTestProject>
+    <IsTestSupportProject>true</IsTestSupportProject>
+  </PropertyGroup>
+
+  <!-- Disables StyleCopAnalyzer, Remove this property to enable it -->
+  <PropertyGroup>
+    <EnableStyleCopAnalyzers>false</EnableStyleCopAnalyzers>
+  </PropertyGroup>
+
+  <!-- Import the common SDK build properties. -->
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.props))\Directory.Build.props" />
+
+  <!-- Set any needed overrides to the decisions made within the common SDK build properties. -->
+  <PropertyGroup>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
+    <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
+    <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
+
+    <!-- Do not inherit implicit dependencies from the engineering system during build or packaging -->
+    <ImportDefaultReferences>false</ImportDefaultReferences>
+
+    <!-- Disable the custom analyzer; it is currently in development and based against draft standards. -->
+    <EnableClientSdkAnalyzers>false</EnableClientSdkAnalyzers>
+
+    <!-- If there was no override specified, assume a project reference to Azure.Messaging.EventHubs -->
+    <UseProjectReferenceToAzureEventHubs Condition="'$(UseProjectReferenceToAzureEventHubs)' == ''">true</UseProjectReferenceToAzureEventHubs>
```

Comment:
Like Core, this will be flipped just prior to a release to ensure a package reference is used.